documentation/restler/QuickStart.md

```diff
@@ -0,0 +1,139 @@
+python ./build-restler.py --dest_dir $restler_bin --python_path /opt/homebrew/bin/python3
+```
+
+## Create the initial Restler config and dict files
+
+You need to create a Restler config file and a dict file for each API you want to test. You can use RESTler to create the initial files for you, with the `generateConfig` command. You need to specify the API spec file(s) -- here we use the Azure Databricks service as an example. 
+
+```sh
+specs=$(find /Users/mikekistler/Projects/Azure/azure-rest-api-specs/specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-04-01-preview -type f -depth 1)
+dotnet $restler_bin/restler/Restler.dll generate_config --specs ${=specs}
+```
+
+This creates a new directory `restlerConfig` with the following files:
+- `annotations.json` - an empty file where you can add annotations for the API spec
+- `config.json` - the Restler config file
+- `dict.json` - a Restler dictionary file
+- `engine_settings.json` - the Restler engine settings file  
+
+## Customize the Restler configuration files
+
+You often need to specify values for variables in the API definition.
+You can do this by adding these values in `restler_custom_payload` property in the `dict.json` file.
+
+For ARM APIs you generally need to specify the subscriptionId, resourceGroupName, and location.
+You may also want to specify the api-version, in case an operation does not have an example value for it.
+You can use the `jq` command to add these values. Here is an example:
+
+```sh
+vars='{
+    "subscriptionId": ["my-subscription-id"],
+    "resourceGroupName": ["my-resource-group"],
+    "location": ["my-location"],
+    "api-version": ["2022-04-01-preview"]
+}'
+jq ".restler_custom_payload = $vars" restlerConfig/dict.json > dict.json
+```
+
+Next update `config.json` to disable data fuzzing.
+
+```sh
+jq '.DataFuzzing = false' restlerConfig/config.json > config.json
+```
+
+No changes are needed to `annotations.json` or `engine_settings.json`, so you can just copy them from the `restlerConfig` directory.
+
+```sh
+cp restlerConfig/annotations.json .
+cp restlerConfig/engine_settings.json .
+```
+
+## Compile tne new configuration
+
+```sh
+dotnet $restler_bin/restler/Restler.dll compile config.json
+```
+
+This creates a new directory `Compile` with many files.
+
+## Set up the authentication script
+
+One last piece to put in place is the script that will supply RESTler with the authentication token.
+For Azure services that accept AAD tokens, you can use the following script (called `getToken.sh`),
+which uses the [Azure CLI][] to get the token.
+
+```sh
+#!/bin/bash
+
+find . -name 'token.json' -depth 1 -mtime -1h | grep . &> /dev/null || az account get-access-token > token.json
+
+token=$(jq -r '.accessToken' token.json)
+
+echo "{'user1':{}, 'user2':{}}"
+echo "Authorization: bearer ${token}"
+echo "Authorization: shadow_unit_test_token"
+```
```

Comment:
Since the second token isn't needed for 'Test' mode, it might be good to simplify this to print just one token.

Reply:
Also remove 'user2' from line 88